### PR TITLE
add reference to zha_event

### PR DESCRIPTION
**Pull request by blakadder**
## Proposed change
This seems to be the most asked question in Home Assistant discord 

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/zha.markdown [hunk 1]
```diff
@@ -53,6 +53,12 @@ There is currently support for the following device types within Home Assistant:
 - Sensor
 - Switch
 
+<div class="note warning">
+
+Remotes use [Events](/docs/events/), they do not create entities! You can subscribe to `zha_event` event type in [Developer Tools/Events](/docs/tools/dev-tools/).
+
+</div>
```
Comment (frenck) [inline source/_integrations/zha.markdown:60]: Two comments:

- Shouldn't these be used via device automations?
- Is it really needed to put up a big warning box at the top of the documentation about this?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/zha.markdown (hunk 1 above)
Comment (blakadder) [reply source/_integrations/zha.markdown:60]: i have no idea, i do not use zha. someone with more knowledge could explain it better
There's a question in Discord on their IKEA whatnot not producing switch entities and then someone answers use `zha_events` which when googled don't give any straightforward link to home assistant docs

Comment (frenck) [reply source/_integrations/zha.markdown:60]: So, this is not just remotes, but general button action in Home Assistant translates to events (at least, they should be).

I have no problem mentioning that and mentioning the event, but in general, using device automation is advised for these things.

But that, of course, is all a different subject as the large warning is needed or not.

Comment (Adminiuga) [reply source/_integrations/zha.markdown:60]: > Shouldn't these be used via device automations

Quirk is needed for the device automation actions as some zha remotes are quite specific.
But even with the device automation, beginners expect entities from the remotes.

Comment (Adminiuga) [reply source/_integrations/zha.markdown:60]: > Is it really needed to put up a big warning box at the top of the documentation about this?

Maybe not a warning, but just a note?

**Review by Adminiuga — APPROVED** [review]
Maybe just should be a note, instead of a warning, but otherwise looks good.

File: source/_integrations/zha.markdown (hunk 1 above)
Comment (frenck) [reply source/_integrations/zha.markdown:60]: My point is, instead of a big box, it can also be a chapter with normal text and context.

**Pull request closed without merging**